tsconfig.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
why do we need a tsconfig.json here?

Reply:
It was added for local testing inside this project. We can safely remove it.  Fixed in 1a44979